### ISSUE TITLE
Q-Select exceed size of label if is large

### ISSUE THREAD

**Issue by brunokunace** [quasarframework/quasar#3491]
### Software version
```
Operating System                                                                Windows_NT(10.0.17134) - win32/x64
NodeJs                                                                          8.9.4

Global packages
  NPM                                                                           5.6.0
  yarn                                                                          1.6.0
  quasar-cli                                                                    0.17.20
  vue-cli                                                                       3.0.0
  cordova                                                                       7.0.1

Important local packages
  quasar-cli                                                                    0.17.20 (Quasar Framework CLI)
  quasar-framework                                                              0.17.17 (Build responsive SPA, SSR, PWA, Hybrid Mobile Apps and Electron apps, all simultaneously using the same codebase)
  quasar-extras                                                                 2.0.8   (Quasar Framework fonts, icons and animations)
  vue                                                                           2.5.17  (Reactive, component-oriented view layer for modern web interfaces.)
  vue-router                                                                    3.0.1   (Official router for Vue.js 2)
  vuex                                                                          3.0.1   (state management for Vue.js)
```

### JsFiddle
https://codepen.io/anon/pen/eXggYK

### What did you get as the error?
When i use select with large name of label, the label exceed max size of field.
When i use overflow: hidden on div for solve previous question, the float-label hidden.

### What were you expecting?

Hidden exceeded size of label and dont hide float-label

### What steps did you take, to get the error?

Select a item in codepen example.

**Comment by Lou-Der** [comment]
I'm also experiencing the overflow of large values in q-selects. This is happening in chrome and IE v17.

The codepen doesn't really give a good example.   However remove the .q-select css and you can see what he's trying to show.

Also, the CSS workaround causes issues with q-tables selected "records per page" not being visible.

Thanks

**Comment by rstoenescu** [comment]
Short note: this has been fixed in v1. It will take a bit to fix it for 0.17 too.

**Comment by brunokunace** [comment]
i can fix with "hack", thanks to @wilcorrea to this.

```
.my-select div.q-input-target {
       overflow: hidden;
       width: 0;
}
```
